parlemonde/1village
# feat(Activity): add new symbol activity & adapt components to display it

### Motivation

To solve this ticket : https://trello.com/c/GGZd7aaH/2-pr%C3%A9senter-un-symbole

### Changes

  - Add new pages to create activity
  - Add new activity type
  - Add new Icon for this activity
  - Update navigation links
  - Modify activity card to display symbol

### Test

 - `git checkout feat/symbol-activity`
 - In navigation component you should see "Présenter un symbole"
 - Click on link, you should be redirect to create symbol
 - Fill the form and try to play with this activity
 - Fill the form until step 3 whithout publish, you should see that draft version is save (go to "mes activités" to verify)
 - Publish and try to answer
 - Check the behavior of the new activity and its card.
